fastlane/Fastfile

```diff
@@ -154,79 +154,132 @@ platform :android do
     # This dependency could be removed with a little more time to tidy up this script to do the branching/merging manually.
 
     desc "Create new release"
-    lane :release do
+    lane :release do |options|
 
         ensure_git_status_clean
         ensure_git_branch( branch: 'develop' )
 
-        newVersion = determine_version_number()
-        releaseNotes = determine_release_notes()
-
-        if UI.confirm("Are you sure you're happy with this release?\n\nVersion=#{newVersion}\nRelease Notes:\n#{releaseNotes}\n")
-          UI.success "Creating release branch for release/#{newVersion}"
-
-          sh "git flow release start #{newVersion}"
+        options_release_number = options[:release_number]
+        options_release_notes = options[:release_notes]
+        options_notes_type = options[:notes_type]
+
+        newVersion = determine_version_number(
+            release_number: options_release_number
+        )
+        releaseNotes = determine_release_notes(
+            release_notes: options_release_notes,
+            notes_type: options_notes_type
+        )
+
+        # We will prompt user input if no release_number is passed OR no notes_type is passed.
+        hasPromptedForUserInput = options_release_number == nil || (options_notes_type == nil && options_notes_type == nil)
+
+        if !hasPromptedForUserInput
```

Comment:
@CDRussell I had to do additional changes to bypass this confirmation when params are passed.